docs/data-access/icgc-25k-data.md

```diff
@@ -0,0 +1,100 @@
+---
+id: icgc-25k-data
+title: ICGC 25K Data Access
+platform_key: DOCS_ICGC_25K
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The ICGC Data Portal was retired in June 2024.
+
+The [Data Portal](https://pubmed.ncbi.nlm.nih.gov/21930502/) served as a hub and reponsitory, a culmination of the collaborative effort between International Cancer Genome Consortium and multiple partners from various cancer projects (including [TCGA](https://www.cancer.gov/ccg/research/genome-sequencing/tcga) and [Sanger Cancer genome project](https://www.sanger.ac.uk/group/cancer-genome-project/)).
+
+The aim was to analyze multiple cancer types and share open access data such as simple somatic mutations, copy number alterations, structural rearrangements, gene expression, microRNAs, DNA methylation and exon junctions.
+
+### Relocated ICGC 25K Data
+
+Although the interactive data portal was shut down, data from the project remains available. The final release data, as well as the PCAWG project data are now hosted by ICGC ARGO. This data is [available to authorized users via an SFTP](#accessing-icgc-25k-release-data).
+
+Files from other contributing projects are all hosted by ICGC's partner repositories. To access this data, you will need to identify which repository hosts the data you are looking for and then request the data through their service. A [mapping file is available](#mapping-icgc-legacy-data-to-external-repositories) to download below which maps ICGC 25K file IDs to their current hosted location.
+
+## Accessing ICGC 25K Release Data
+
+An SFTP server is available to access ICGC Release Data and PCAWG data.
+
+The server hosts three data directories with the following data:
+
+- `/release_28` - This is the Data Portal data Release 28 of the International Cancer Genome Consortium (ICGC).
+- `/PCAWG` - Analysis results from the PCAWG study.
+- `/Supplemental` - Corrected clinical metadata and RNA-Seq raw read counts for projects LICA-FR and PRAD-UK.
+
+Only users with DACO approval are able to access the SFTP server. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).
```

Comment:
Only users with DACO approval are able to access the SFTP server. -> The SFTP server is available to authorized, DACO-approved users only.